content/installation/netcore/install/Rules-netcore.md

```diff
@@ -0,0 +1,74 @@
+
+<!--
+title: "Contrast .NET Core Agent Rules Support"
+description: "Assess and Protect rules supported by the Contrast .NET Core agent"
+tags: "installation agent .NET Core windows assess protect rules"
+-->
+
+
```

Comment:
@ldemidov and @DanFiedler 
I moved these to their own article. We can keep it on the page with Supported Technologies, as it was, or create a new Policy page to mirror the .NET docs. Either way is fine by me.

Reply:
I think keeping it on Supported Technologies is ok for now.

I think we don't need a Policy page for netcore yet.  It's not tested and we may want to pare that page down to just file analysis rules anyway.  I'm curious what Dan thinks about that.